### DISCUSSION TITLE
Weird error with PeerTube videos over a gigabyte

### DISCUSSION THREAD

**Discussion by dhk2** [seaweedfs/seaweedfs#5488] [General]
I've built a few SeaweedFS object stores for PeerTube instances and it's been working perfectly. For the last few months I've been experiencing an issue with longer videos getting a BucketAlreadyExists error when trying to upload to PeerTube instances using the S3 object storage feature with SeaweedFS. Always getting the error on videos over a gig.

Last test was creating a bucket on AWS and verifying it works there with no errors. It may still be an issue with PeerTube that AWS handles more gracefully, but it's starting to look like a SeaweedFS issue. 

Any suggestions on what it may be, or where to look for info or hints on what's going on?

**Comment by chrislusf** [comment]
please make a reproducible case, to reproduce and to confirm the fix.

**Comment by kmlebedev** [comment]
You need to create a docker compose file that will run seaweedfs and PeerTube and run commands in the terminal to reproduce the problem

https://joinpeertube.org/faq#what-are-the-peertube-features-for-administrators
